client/schematic.go

```diff
@@ -1,33 +1,34 @@
-package schematic
+package client
 
 import (
 	"context"
 	"time"
 
-	"github.com/SchematicHQ/schematic-go/api"
+	schematicgo "github.com/schematichq/schematic-go"
+	core "github.com/schematichq/schematic-go/core"
 )
 
-type Client interface {
+type SchematicClient interface {
 	// Check a flag to get a boolean value
-	CheckFlag(context.Context, *CheckFlagRequestBody, string) bool
+	CheckFlag(context.Context, *schematicgo.CheckFlagRequestBody, string) bool
 
 	// Identify a user
-	Identify(context.Context, *EventBodyIdentify)
+	Identify(context.Context, *schematicgo.EventBodyIdentify)
 
 	// Track an action
-	Track(context.Context, *EventBodyTrack)
+	Track(context.Context, *schematicgo.EventBodyTrack)
 
 	// Access bindings for Schematic API
-	API() *api.APIClient
+	API() *Client
```

Comment:
@bpapillon thinking about this more we can make this so that we don't need to call `API()` before calling all of the api funcs. Because we're not mocking `SchematicClient` anymore we can make `SchematicClient` a `struct` and therefore call the funcs directly on  a `SchematicClient`